interceptors-opentelemetry/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "interceptors-opentelemetry-example",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "build": "tsc --build src/worker/tsconfig.json",
```

Comment:
Can we update these scripts to use nodemon?

Reply:
`hello-world-mtls` only uses it for `start.watch`—should it be used in others as well?
 
![image](https://user-images.githubusercontent.com/251288/132592374-857c92f6-3a84-4c8e-9cc7-e2f0018f22e2.png)

Reply:
Yeah I think adding start.watch to the other samples would be great

Reply:
Agree